### PR TITLE
Refactor backend code

### LINKED ISSUE #43: Refactor backend code

### PR DESCRIPTION
**Pull request by Dogacel**
Closes #43

### CODE REVIEW

File: api/flaskr/routes/image.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+@image_route.route('/uploadProductImage', methods=(['POST']))
+def uploadProductImage():
+    file = request.files.get('image')
+    fashion_image_collection = mongo.fashionImages.db.images
+    mongo.fashionImages.save_file(file.filename, file)
+    fashion_image_collection.insert({'photo_name': file.filename})
+    # Return more meaningful data maybe
+
+    return 'Done!'
```
Comment (AtaAtasoy) [inline api/flaskr/routes/image.py]: IMO, a Response object with a status code or a JSON response would be better.

File: api/flaskr/routes/auth.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    login_user = users.find_one({'username': username})
+    if login_user:
+        if bcrypt.check_password_hash(login_user['password'], password):
+            session['username'] = username
+            expires = datetime.timedelta(hours=1)
+            tokens = {
+                'access_token': create_access_token(identity=username, expires_delta=False),
+                'refresh_token': create_refresh_token(identity=username)
+            }
+            return jsonify(tokens), 200
```
Comment (AtaAtasoy) [inline api/flaskr/routes/auth.py]: is it possible to edit this line as ?
```python
return Response(tokens, status=200)
```
or 
```python
return Response(jsonify(tokens), status=200)
```
I'm not sure which one of the two lines of code would work, thus I suggested both. I think you get the idea 😄 

I also think returning a Response object in each of the routes would make our code more consistent.

File: api/flaskr/routes/auth.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    return jsonify({"msg": "Successfully logged out"}), 200
+
+
+# Endpoint for revoking the current users refresh token
+@auth_route.route('/logout2', methods=['DELETE'])
+@jwt_refresh_token_required
+def logout2():
+    jti = get_raw_jwt()['jti']
+    blacklist.add(jti)
+    return jsonify({"msg": "Successfully logged out"}), 200
```
Comment (AtaAtasoy) [inline api/flaskr/routes/auth.py]: Is it possible to wrap this return value around Response() object ? Maybe we should change one of the return values and test it before we change every single response with Response().

**Review by AtaAtasoy — CHANGES_REQUESTED** [review]

**Comment by Dogacel** [comment]
@AtaAtasoy I removed 200 because it is the default status for a response object.

**Review by AtaAtasoy — APPROVED** [review]
Looks good. Eline saglik 😸

**Pull request merged**